src/Elliptic/Systems/IrrotationalBns/Tags.hpp

```diff
@@ -0,0 +1,89 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+/// \file
+/// Defines DataBox tags for the Poisson system
+
+#pragma once
+
+#include <string>
+
+#include "DataStructures/DataBox/Tag.hpp"
+#include "DataStructures/Tensor/TypeAliases.hpp"
+
+/// \cond
+class DataVector;
+/// \endcond
+
+/*!
+ * \ingroup EllipticSystemsGroup
+ * \brief Items related to solving for irrotational bns initial data
+ */
+namespace IrrotationalBns::Tags {
+
+/*!
+ * \brief The shift plus a spatial vector \f$ k^i\f$
+ * \f$B^i = \beta^i + k^i\f$
+ */
+template <typename DataType>
+struct RotationalShift : db::SimpleTag {
+  using type = tnsr::I<DataType, 3>;
+};
+/*!
+ * \brief The stress-energy corresponding to the rotation shift
+ *
+ *
+ * \f[\Sigma^i_j = \frac{1}{2}\frac{B^iB_j}{\alpha^2}\f]
+ */
+template <typename DataType>
+struct RotationalShiftStress : db::SimpleTag {
+  using type = tnsr::II<DataType, 3>;
+};
+/*!
+ * \brief  The derivative  \f$D_i \ln (\alpha/h)\f$
+ */
+template <typename DataType>
+struct DerivLogLapseOverSpecificEnthalpy : db::SimpleTag {
+  using type = tnsr::i<DataType, 3>;
+};
+
+/*!
+ * \brief The velocity potential for the fluid flow \f$\Phi\f$, i.e. the
+ * curl-free part of the fluid is given by \f$\nabla_a \Phi = h u_a\f$
+ */
+template <typename DataType>
+struct VelocityPotential : db::SimpleTag {
+  using type = Scalar<DataType>;
+};
+/*!
+ * \brief The auxiliary velocity variable that represents
+ * the flux of the velocity potential
+ * \f[
+ *  U_i \equiv D_i \Phi  - \frac{C + B^jD_j \Phi}{\alpha^2}B_i
+ * \f]
+ */
+template <typename DataType>
+struct AuxiliaryVelocity : db::SimpleTag {
```

Comment:
You can delete this tag, and also some or all of the tags below this (at least for now).

Reply:
So basically If I deleted these now they would get added back in when the executable is added?

Reply:
You can keep the ones you surely will need, like the Euler constant. But e.g. the `FixedSources` tag you don't need because there's already one in `DataStructures/DataBox/Prefixes.hpp`.